### PR TITLE
 Created a text index & changed search queries

### PR DESCRIPTION
**Pull request by YaroslavSemeniuk**
Git-2582 created a text index and changed search queries for full text search

### CODE REVIEW

File: utilities/operations/search/searchWobjects.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable camelcase */
-const { FIELDS_NAMES, SEARCH_FIELDS, OBJECT_TYPES } = require('constants/wobjectsData');
+const { FIELDS_NAMES, OBJECT_TYPES } = require('constants/wobjectsData');
```
Comment (Waiviogit) [inline utilities/operations/search/searchWobjects.js:2]: you have removed search fields and do not create index on that fields in search

File: utilities/operations/search/searchWobjects.js [hunk 2]
```diff
@@ -140,6 +140,9 @@ const makeSitePipeline = ({
         priority: { $cond: { if: { $eq: ['$parent', forParent] }, then: 1, else: 0 } },
       },
     }, { $sort: { priority: -1, [sort]: -1 } });
+  }
+  if (string !== '') {
```
Comment (Waiviogit) [inline utilities/operations/search/searchWobjects.js:144]: if (string)

File: utilities/operations/search/searchWobjects.js [hunk 3]
```diff
@@ -159,6 +162,8 @@ const makePipeline = ({
       },
     },
     { $sort: { crucial_wobject: -1, priority: -1, weight: -1 } });
+  } else if (string !== '') {
```
Comment (Waiviogit) [inline utilities/operations/search/searchWobjects.js:165]: if (string)

File: utilities/operations/search/searchWobjects.js [hunk 4]
```diff
@@ -210,6 +215,14 @@ const matchSitesPipe = ({
       },
     });
   }
+  if (string !== '') {
```
Comment (Waiviogit) [inline utilities/operations/search/searchWobjects.js:218]: if (string)

**Review by Fesmofet — APPROVED** [review]

**Comment by YaroslavSemeniuk** [comment]
searches can be performed by the following fields: name, email, phone, address, author_permlink

**Pull request merged**